### ISSUE TITLE
webdriverio@5.20.1 breaks compatibility with Node 8

### ISSUE THREAD

**Issue by fafnirical** [webdriverio/webdriverio#5094]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.20.1
 - **Mode:** N/A
 - **If WDIO Testrunner, running sync/async:** N/A
 - **Node.js version:** 8.17.0
 - **NPM version:** yarn 1.6.0
 - **Browser name and version:** N/A
 - **Platform name and version:** N/A
 - **Additional wdio packages used (if applicable):** N/A

**Config of WebdriverIO**
N/A

**Describe the bug**
`webdriverio@5` declares a minimum Node dependency of Node 8.11.0 (https://github.com/webdriverio/webdriverio/blob/v5.20.1/packages/webdriverio/package.json#L17) or Node 8.11.2 (https://webdriver.io/docs/gettingstarted.html#taking-the-first-step).
However, `webdriverio@5.20.1` can't be installed on Node 8, due to its `serialize-error@^6.0.0` dependency. `serialize-error@^6.0.0` has a dependency on Node >=10: https://github.com/sindresorhus/serialize-error/blob/v6.0.0/package.json#L14

**To Reproduce**
1.  Try to install `webdriverio@5.20.1` on Node 8
    ```sh
    nvm install 8.17.0
    nvm use 8.17.0

    yarn add --dev webdriverio@5.20.1
    ```

**Expected behavior**
`webdriverio@5` maintains compatibility with Node 8.

**Additional context**
This appears to have been broken by https://github.com/webdriverio/webdriverio/commit/c821564f82216431b1b72486fc6cfd87cc248484. See also https://github.com/webdriverio/webdriverio/issues/4360 and https://github.com/webdriverio/webdriverio/issues/4356.

**Comment by christian-bromann** [comment]
@fafnirical thanks for reporting. I reverted the change an re-released as `v5.20.2`. This should fix it.

**Comment by fafnirical** [comment]
Thanks for the quick turnaround! 😃
